### ISSUE TITLE
SVT: Windows VSCode: Icons not showing up in 0.8.0

### ISSUE THREAD

**Issue by naveenkaratekid** [eclipse/codewind#1707]
**Codewind version:0.8.0**
**OS:Windows**

**IDE extension version:0.8.0**
**IDE version: 1.41.:**

**Description:**
In VSCode, I cannot see any icons for Codeiwnd 0.8.0. All I see is the alt text for them. Also, some of the alt text in the Manage Templates page is hard to read because the text is black in a dark background, making it difficult to determine if I have disabled or enabled a specific template source.

![image](https://user-images.githubusercontent.com/19738071/72170530-07173700-339f-11ea-8e53-b521f60176fa.png)

@jagraj

**Comment by tetchel** [comment]
i will also make the alt text for the Remove button the name instead of the description 😢

**Comment by tetchel** [comment]
@naveenkaratekid Should be fixed now

**Comment by naveenkaratekid** [comment]
@tetchel verified that it's fixed. Closing the issue now.
